src/components/UserInfo.js

```diff
@@ -12,7 +13,11 @@ export default class UserInfo {
   }
 
   setUserInfo(userData) {
-    this._nameElement.textContent = userData.title;
-    this._jobElement.textContent = userData.description;
+    this._nameElement.textContent = userData.name;
+    this._jobElement.textContent = userData.about;
+  }
+
+  setUserAvatar(avatar) {
```

Comment:
It;s great to have a setter for the avatar, since it's controlled by a different form